vhdbuilder/packer/post-install-dependencies.sh

```diff
@@ -15,7 +15,7 @@ VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
 PERFORMANCE_DATA_FILE=/opt/azure/vhd-build-performance-data.json
 
 # Hardcode the desired size of the OS disk so we don't accidently rely on extra disk space
-MAX_BLOCK_COUNT=30298176 # 30 GB
+MAX_BLOCK_COUNT=60596352 # 60 GB
```

Comment:
unless we know we can safely increase this for _all_ SKUs, we should make this GB200-specific